src/gpu/sycl/sycl_primitive_conf.hpp

```diff
@@ -127,6 +127,25 @@ struct sycl_shuffle_conf_t {
     dim_t work_amount;
 };
 
+struct sycl_reorder_conf_t {
+    xpu::sycl::md_t src_md;
+    xpu::sycl::md_t dst_md;
+    xpu::sycl::md_t scales;
+
+    bool do_scale_src;
+    int scale_src_mask;
+    bool do_scale_dst;
+    int scale_dst_mask;
+
+    int ndims;
+
+    int block_size;
+    int wg_size;
+    int wk_size;
```

Comment:
Mind to comment the usage of these members?

Reply:
These were copied from binary primitive. Personally I am not a fan of how this is done and do not have that great understanding of that, but I see value of keeping implementations consistent between primitives.

Reply:
We probably will at one point, but currently that is not high priority.